examples/downstream-v0.sh

```diff
@@ -3,14 +3,18 @@
 # This script will run a local OLMv0 downstream operation, to allow someone to
 # debug a failure.
 #
-# Assumptions:
-# * The tooling directory is located in ${HOME}/git/openshift/operator-framework-tooling
-# * The v0 executable has been built in there via `make`
-#
 # Call with your github user name, if it's not the same as your current user name.
 #
+# Usage examples:
+#     downstream-v1.sh octocat "-drop-commits fca12345678" /Users/octocat/downstreamv0
+#     downstream-v1.sh octocat "" /Users/octocat/downstreamv0
+#     downstream-v1.sh octocat
 
 GITHUB_USER=${1:-${USER}}
+SYNC_ARGS=${2:-""}
+SYNC_DIR=${3:-$PWD}
+
+TOOLS_REPO_DIR=$(dirname "$(dirname "$0")")
```

Comment:
So it can be set outside of the script.
```suggestion
TOOLS_REPO_DIR=${TOOLS_REPO_DIR:-$(dirname "$(dirname "$0")")}
```

Reply:
This variable will be set to an absolute path of the root of the repo (see the updated PR). This can be one of the options, but for now I'm leaving it outside of the scope of this PR (I expanded it already quite a bit).